kornia/contrib/distance_transform.py

```diff
@@ -54,6 +54,7 @@ def distance_transform(
 
     # It is possible to avoid cloning the image if boundary = image, but this would require modifying the image tensor.
     boundary = image.clone()

```

Comment:
We should evaluate wether this `.clone()` is needed

Reply:
I think it would be better to keep the `.clone()` here. 
If removed, the following loop would change the pixel values of image gradually, which IMO is not a desired behaviour.